p2p/module.go

```diff
@@ -116,13 +116,15 @@ func (m *p2pModule) GetModuleName() string {
 }
 
 func (m *p2pModule) Start() error {
-	logger.Global.Info().Msg("Starting network module")
+	m.logger = logger.Global.CreateLoggerForModule(m.GetModuleName())
+	m.logger.Info().Msg("Starting network module")
 
 	addrbookProvider := getAddrBookProvider(m)
 	currentHeightProvider := getCurrentHeightProvider(m)
 
 	cfg := m.GetBus().GetRuntimeMgr().GetConfig()
 
+	// TODO: pass down logger
```

Comment:
```suggestion
	// TODO: propogate `p2pModule.logger` to the underlying network
```

Reply:
This TODO is now redundant since the logger is created inside the network components.

@bryanchriswhite, in your upcoming PR, would you mind simply removing this TODO? 🙏